### PR TITLE
make modules opt-out after import

### PR DESCRIPTION
**Pull request by nazarewk**
fixes https://github.com/astro/microvm.nix/issues/61

### CODE REVIEW

**Comment by nazarewk** [comment]
this is not side-effect-free, but good enough (only enables imported minimal profile).

File: nixos-modules/microvm/stage-1.nix [hunk 1]
```diff
@@ -69,4 +70,5 @@ in {
     fsType = "tmpfs";
     options = [ "size=50%,mode=0755" ];
   };
+};
```
Comment (astro) [inline nixos-modules/microvm/stage-1.nix]: Lines in between these curly braces need to be indented.

File: nixos-modules/microvm/system.nix [hunk 2]
```diff
@@ -90,4 +92,5 @@ in
       generators = { systemd-gpt-auto-generator = "/dev/null"; };
     };
 
+};
```
Comment (astro) [inline nixos-modules/microvm/system.nix]: Likewise

**Review by astro — CHANGES_REQUESTED** [review]
Thank you. This looks good to me except for the indentation.

May I ask about the use-case for this change? I want to avoid breaking it in the future.

**Comment by nazarewk** [comment]
I thought about formatting it with `nixpkgs-fmt`, but it made MANY changes.

About my use case: I have a [repo holding all my configs in opt-in modules](https://github.com/nazarewk-iac/nix-configs), so i can pick only relevant pieces for different configs/machines. It works fine except for things using "plain" configs not hidden behind (eg: [this importing qemu guest from nixpkgs](https://github.com/nazarewk-iac/nix-configs/blob/b8f5532608650f57f1f14ecd05a0e9b07887d088/machines/hetzner/wg-0/default.nix#L2-L6))

**Comment by astro** [comment]
git's word-diff didn't work well for me regardless of the chosen algorithm. Therefore I indented myself.

Thank you!

**Pull request closed without merging**